testmon/testmon_core.py

```diff
@@ -118,7 +119,10 @@ def eval_variant(run_variant, **kwargs):
     if not run_variant:
         return ''
 
-    eval_globals = {'os': os, 'sys': sys}
+    def md5(s):
+        return hashlib.md5(s.encode()).hexdigest()
```

Comment:
I wanted to have this quite restricted, because I feel the run_variant could be replaced by something else.